src/Components/Table/index.js

```diff
@@ -39,35 +43,38 @@ function Table({ tableKey, customClass, addMainInput, centerInput, main }) {
       <div className='boxes'>
         <div onClick={onClickSpan} className='hide-input first-first'>
           <span
+            id={`box-${tableKey}-0`}
             contentEditable={true}
             onPaste={onPreventStylePaste}
             data-index={0}
             className={`box ${customClass}`}
-            onBlur={main ? onChangeMainInput : null}
+            onBlur={main ? onChangeMainInput : onChangeSubInput}
             autoComplete='off'
             data-placeholder={main ? '목표 1' : ''}
             onKeyDown={onPreventShiftTab}
-          />
+          ></span>
```

Comment:
I think self-closing tag could be better because there is nothing between them! 😁
how do you think of this?